sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/ChangeFeedContextClient.java

```diff
@@ -146,4 +150,18 @@
      * Closes the document client instance and cleans up the resources.
      */
     void close();
+
+    /**
+     * Gets the internal {@link Scheduler} that hosts a pool of ExecutorService-based workers for any change feed processor related tasks.
+     *
+     * @return a {@link Scheduler} that hosts a pool of ExecutorService-based workers..
+     */
+    Scheduler getScheduler();
```

Comment:
I feel we should start these as Beta APIs, thoughts @milismsft ?

Reply:
Just checked, you already marked them as Beta in public surface, good :)

Reply:
yep, these are internal (see the implementation in the name space), don't have to be "beta" annotated.